Jenkinsfile_k8s

```diff
@@ -193,7 +193,9 @@ pipeline {
 
                 // Execute build only for this matrix cell's setup
                 retry(count: 2, conditions: [kubernetesAgent(handleNonKubernetes: true), nonresumable()]) {
-                  sh 'packer build -timestamp-ui -force -only="${PKR_VAR_image_type}.${PKR_VAR_agent_os_type}" ./'
+                  infra.withDockerCredentials {
+                    sh 'packer build -timestamp-ui -force -only="${PKR_VAR_image_type}.${PKR_VAR_agent_os_type}" ./'
+                  }
```

Comment:
packer without credentials, then if tag name & branch is primary, docker push with credentials